source/exec.tex

```diff
@@ -0,0 +1,4872 @@
+  }
+}
+\end{codeblock}
+
+\pnum
+The expression \tcode{connect(sndr, rcvr)} is expression-equivalent to:
+\begin{itemize}
+\item
+\tcode{new_sndr.connect(rcvr)} if that expression is well-formed.
+
+\mandates
+The type of the expression above satisfies \libconcept{operation_state}.
+
+\item
+Otherwise, \tcode{\exposid{connect-awaitable}(new_sndr, rcvr)}.
+\end{itemize}
+
+\mandates
+\tcode{\libconcept{sender}<Sndr> \&\& \libconcept{receiver}<Rcvr>} is \tcode{true}.
+
+\rSec2[exec.factories]{Sender factories}
+
+\rSec3[exec.schedule]{\tcode{execution::schedule}}
+
+\pnum
+\tcode{schedule} obtains a schedule sender\iref{exec.async.ops}
+from a scheduler.
+
+\pnum
+The name \tcode{schedule} denotes a customization point object.
+For a subexpression \tcode{sch},
+the expression \tcode{schedule(sch)} is expression-equivalent to
+\tcode{sch.schedule()}.
+
+\pnum
+\mandates
+The type of \tcode{sch.schedule()} satisfies \libconcept{sender}.
+
+\pnum
+If the expression
+\begin{codeblock}
+get_completion_scheduler<set_value_t>(get_env(sch.schedule())) == sch
+\end{codeblock}
+is ill-formed or evaluates to \tcode{false},
+the behavior of calling \tcode{schedule(sch)} is undefined.
+
+\rSec3[exec.just]{\tcode{execution::just}, \tcode{execution::just_error}, \tcode{execution::just_stopped}}
+
+\pnum
+\tcode{just}, \tcode{just_error}, and \tcode{just_stopped} are sender factories
+whose asynchronous operations complete synchronously in their start operation
+with a value completion operation,
+an error completion operation, or
+a stopped completion operation, respectively.
+
+\pnum
+The names \tcode{just}, \tcode{just_error}, and \tcode{just_stopped} denote
+customization point objects.
+Let \exposid{just-cpo} be one of
+\tcode{just}, \tcode{just_error}, or \tcode{just_stopped}.
+For a pack of subexpressions \tcode{ts},
+let \tcode{Ts} be the pack of types \tcode{decltype((ts))}.
+The expression \tcode{\exposid{just-cpo}(ts...)} is ill-formed if
+\begin{itemize}
+\item
+\tcode{(\exposid{movable-value}<Ts> \&\&...)} is \tcode{false}, or
+\item
+\exposid{just-cpo} is \tcode{just_error} and
+\tcode{sizeof...(ts) == 1} is \tcode{false}, or
+\item
+\exposid{just-cpo} is \tcode{just_stopped} and
+\tcode{sizeof...(ts) == 0} is \tcode{false}.
+\end{itemize}
+
+Otherwise, it is expression-equivalent to
+\tcode{\exposid{make-sender}(\exposid{just-cpo}, \exposid{product-type}{ts...})}.
+
+For \tcode{just}, \tcode{just_error}, and \tcode{just_stopped},
+let \exposid{set-cpo} be
+\tcode{set_value}, \tcode{set_error}, and \tcode{set_stopped}, respectively.
+The exposition-only class template \exposid{impls-for}\iref{exec.snd.general}
+is specialized for \exposid{just-cpo} as follows:
+\begin{codeblock}
+namespace std::execution {
+  template<>
+  struct @\exposid{impls-for<}@@\exposid{decayed-typeof}@<@\exposid{just-cpo}@>> : @\exposid{default-impls}@ {
+    static constexpr auto start =
```

Comment:
```suggestion
  struct @\exposid{impls-for}@<@\exposid{decayed-typeof}@<@\exposid{just-cpo}@>> : @\exposid{default-impls}@ {
    static constexpr auto @\exposid{start}@ =
```

Reply:
The first issue is fixed (obviously a transcription problem).
The second issue is also fixed, but was already present in the incoming paper.